__tests__/app.test.js

```diff
@@ -156,3 +157,53 @@ describe('GET /api/articles/:article_id/comments', () =>{
     }) 
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
```

Comment:
Would also be good to have a 201 test for when unnecessary properties are on the comment body.
(they should be ignored!)

Reply:
I am on it!